internal/cmdget/cmdget.go

```diff
@@ -87,20 +87,15 @@ func (r *Runner) preRunE(_ *cobra.Command, args []string) error {
 			args[1] = resolvedPath
 		}
 	}
-	destination, err := parse.ParseArgs(r.ctx, args, parse.Options{
-		SetGit: func(git *kptfilev1.Git) error {
-			r.Get.Git = git
-			r.Get.Upstream = remote.NewGitUpstream(git)
-			return nil
-		},
-		SetOci: func(oci *kptfilev1.Oci) error {
-			r.Get.Upstream = remote.NewOciUpstream(oci)
-			return nil
-		},
-	})
+
+	upstream, destination, err := parseref.ParseArgs(
+		args,
+		location.WithContext(r.ctx),
```

Comment:
So do we store anything location specific in the `ctx` here ? This is more of a note I may find the answer later in the PR.